deployment/deployment.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: default
+title: Deploying RAPIDS
+nav_order: 5
+permalink: deployment
+---
+
+# Deploying RAPIDS
+{: .no_toc}
+
+RAPIDS libraries are deployable across multiple environments types:
+1. Single Node Single GPU
+2. Single Node Multiple GPUs
+3. Multiple Nodes, Multiple GPUs
+4. Multiple or Single Node managed by a scheduling service:
+    a. Kubernetes
+    b. SLURM
+
+Included below are guides for deploying to some of these environments.
+{: .fs-6 .fw-300}
+
+### Single Node Single GPU
+In Single Node, Single GPU environments simply [install](https://rapids.ai/start.html#get-rapids) and import the libraries:
+```
+>>> import cudf, cuml, cugraph, cuspatial # Good to go!
+```
+<!--
+Modify the codeblock if needed
+-->
+{: .mb-7 }
+
+### Single Node Multi GPU
+RAPIDS libraries also scale up to multiple GPUs on a single node. This is most easily setup using [Dask-CUDA](https://github.com/rapidsai/dask-cuda)'s `LocalCUDACluster` API to detect and start one worker process per available GPU.
+```
+>>> from dask_cuda import LocalCUDACluster
+>>> from dask.distributed import Client
+>>> import dask_cudf
+
+>>> IPADDR = "0.0.0.0" # Change to more specific node IP if desired
+>>> cluster = LocalCUDACluster(ip= IPADDR)
+>>> client = Client(clsuter)
+```
+{: .mb-7 }
+<!--
+Include output of client maybe?
+Choose a beginner example instead of intermediate?
+-->
+### Multi Node Multi GPU
+
+#### **Manual Setup**
+This is the most basic means of deploying RAPIDS to multiple hosts. Most more advanced deployments use the same commands with environment configuration & parameter variations.
+- Run the `dask-scheduler` on one node
+    ```
+    $ dask-scheduler
+    ```
+- Run `dask-cuda-worker` on each worker node, providing the scheduler node's address. `dask-cuda-worker` will detect available GPUs and fork, starting one dask worker process per GPU.
+
+    ```
+    $ dask-cuda-worker 127.0.0.1:8786
+    Start worker at:   tcp://127.0.0.1:36124
+    Registered to: tcp://127.0.0.1:8786
+    ```
+- Connect to the cluster from a Python script
+    ```
+    >>> from dask.distributed import Client
+    >>> # wrap Dask client API use in a main check
+    >>> if __name__ == "__main__":
+    ...     client = Client(address="127.0.0.1:8786")
+    ...     # use_dask_APIs()
+    ```
+    Notes:
+    - For more information on Dask CLI options use `dask-scheduler --help` and `dask-cuda-worker --help`
+
+    More information on deploying manually can be found [here](https://docs.dask.org/en/latest/setup/cli.html).
+
+    Dask also supports other options like [SSH](https://docs.dask.org/en/latest/setup/ssh.html) and [NFS](https://docs.dask.org/en/latest/setup/hpc.html#using-a-shared-network-file-system-and-a-job-scheduler).
```

Comment:
```suggestion
Dask also supports other options like [SSH](https://docs.dask.org/en/latest/setup/ssh.html) and [NFS](https://docs.dask.org/en/latest/setup/hpc.html#using-a-shared-network-file-system-and-a-job-scheduler):
```
# starts one worker process per GPU on the supplied host list
dask-ssh 10.0.0.1 10.0.0.2 --remote-dask-worker dask_cuda.dask_cuda_worker
```
```

Reply:
The suggestion formatting got messed up, but you should include basic examples of SSH and NFS dask-cuda cluster creation.